com.unity.render-pipelines.universal/Runtime/Passes/TransparentSettingsPass.cs

```diff
@@ -31,12 +30,16 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
             CommandBuffer cmd = CommandBufferPool.Get();
             using (new ProfilingScope(cmd, m_ProfilingSampler))
             {
+                ShadowData shadowData = renderingData.shadowData;
+                int cascadesCount = shadowData.mainLightShadowCascadesCount;
+
+                bool shouldReceiveNoCascades = m_shouldReceiveShadows && cascadesCount == 1;
+                bool shouldReceiveCascades = m_shouldReceiveShadows && cascadesCount > 1;
```

Comment:
very minor suggestion to make this more readable: name receiveShadowsNoCascade and receiveShadowsCascades.

Reply:
I applied it